code/game/antagonist/station/changeling.dm

```diff
@@ -4,7 +4,7 @@
 	role_text_plural = "Changelings"
 	bantype = "changeling"
 	feedback_tag = "changeling_objective"
-	restricted_jobs = list("AI", "Cyborg", "Head of Security", "Captain", "Internal Affairs Agent")
+	restricted_jobs = list("AI", "Cyborg", "Head of Security", "Captain", "Internal Affairs Agent", "Chief Engineer", "Research Director", "Chief Medical Officer", "Head of Personnel")
```

Comment:
Please Remove "Internal Affairs Agent" from those lists, as we no longer have them.